### PR TITLE
File upload

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. 

I am not totally happy with the code, I would like to refactor and add more unit tests, but I think we are taking too long on this task. Maybe we should add a card for refactoring in our Debt Reduction epic.

Thanks!

### CODE REVIEW

File: test/e2e/storage/storage.cases.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
 StorageSelectorScenarios = require('./cases/storage-selector.js');
+NewFolderModalScenarios = require('./cases/new-folder.js');
```
Comment (alex-deaconu) [inline test/e2e/storage/storage.cases.js]: @ezequielc I'm assuming this was missed in the New Folder PR? Thanks.

Comment (ezequielc) [inline test/e2e/storage/storage.cases.js]: @alex-deaconu Exactly, I missed adding it to the commit. Cheers

File: web/partials/storage/storage-modal.html [hunk 2]
```diff
@@ -21,14 +21,69 @@ <h1 id="title" translate="storage-client.select-{{storageFactory.selectorType}}"
...
-	      Upload
+	    <label id="uploadButton" for="upload-files" class="btn btn-lg btn-default" ng-hide="statusDetails.code!==200 || fileListStatus.code===404" ng-disabled="isTrashFolder()">
+        <span translate>storage-client.upload-files</span> 
 	      <i class="fa fa-cloud-upload fa-lg icon-right"></i> 
-	    </button>
+	    </label>
 	  </div>
+
+<div ng-controller="UploadController">
+  <div class="content-box animated fadeInDown upload-panel" ng-show="uploader.queue.length > 0">
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: @ezequielc would you be against splitting this out into its own partial? Just to keep the size of this file manageable. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good :+1: . Yes, I do see room for refactoring, so agreed re: card. However, I would keep that card in the Storage epic. Thanks.

File: web/partials/storage/storage-modal.html (hunk 2 above)
Comment (ezequielc) [inline web/partials/storage/storage-modal.html]: @alex-deaconu Makes sense. I will split it. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu I split the upload panel into its own partial. Please review. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc looks good. Thanks for fixing. :+1:

**Pull request merged**